src/components/user/avatar/UserAvatar.tsx

```diff
@@ -60,36 +61,57 @@ export default class UserAvatar extends React.Component<Props, State> {
   };
 
   public render() {
-    const { accessoryIcon, size } = this.props;
-    const { user } = this.state;
+    const { accessoryIcon, size, icon } = this.props;
     const style = computeStyleSheet();
-    const userInitials = Utils.buildUserInitials(user);
-    const userName = Utils.buildUserName(user);
-    const isNameHyphen = userName === Constants.HYPHEN;
-    const userImageURI = user ? (isNameHyphen ? noPhoto : user.image) : noPhoto;
-    return (
-      <View>
-        {userImageURI ? (
+    // If the icon is provided, we display it instead of the user
+    if (icon) {
```

Comment:
The check should not be based on the icon

Reply:
If there is no icon, we don't display it , no ?

Reply:
This is a selectable icon, it is related to the issue with the 'selected' flag above ;-)

Reply:
I know, but I have changed for your first proposition

Reply:
So I don't understand what to change :p

Reply:
To replace icon with 'selectable' and hardcode the icon

Reply:
Done, I put isSelected instead, because in the modal all items are selectable

Reply:
I tag you @LucasBrazi06 because, weirdly, the status is still requested changes in the PR list preview